charts/adoption-web/values.yaml

```diff
@@ -20,7 +20,7 @@ nodejs:
     IDAM_API_URL: 'https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/o/token'
     REDIS_HOST: 'adoption-web-session-storage-{{ .Values.global.environment }}.redis.cache.windows.net'
     SECURE_COOKIE: 'true'
-    REDIS_PORT: 6380
+    REDIS_PORT: 6379
```

Comment:
Please do not change this value ("in charts folder") as this doesn't correspond to local. It corresponds to the higher environments